coderd/coderdtest/swaggerparser.go

```diff
@@ -297,6 +297,7 @@ func assertPathParametersDefined(t *testing.T, comment SwaggerComment) {
 func assertSecurityDefined(t *testing.T, comment SwaggerComment) {
 	if comment.router == "/updatecheck" ||
 		comment.router == "/buildinfo" ||
+		comment.router == "/experiments" ||
```

Comment:
I'm curious if it doesn't leak the server configuration to the wild audience. It might be easier to spot vulnerable servers with beta features enabled.

Reply:
This is a completely valid point, and I'm open to simply requiring simply an authenticated user on this endpoint.